### PR TITLE
[prometheus-mysql-exporter] Add securityContext and podSecurityContext

### PR DESCRIPTION
**Pull request by ryjogo**
#### What this PR does / why we need it:
Adds securityContext and podSecurityContext to be able to run the image as a `nobody` user

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-mysql-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for prometheus mysql exporter with cloudsqlproxy
 name: prometheus-mysql-exporter
-version: 1.0.2
+version: 1.0.3
```
Comment (monotek) [inline charts/prometheus-mysql-exporter/Chart.yaml:4]: new feature should raise minor version.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-mysql-exporter/Chart.yaml (hunk 1 above)
Comment (ryjogo) [reply charts/prometheus-mysql-exporter/Chart.yaml:4]: correct, thanks.

Comment (monotek) [inline charts/prometheus-mysql-exporter/Chart.yaml:4]: New feature should raise minor version.

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Please sign your commits to get the dco check passed.

**Comment by ryjogo** [comment]
Apologies, forgot to sign.. should be ok now.

**Review by monotek — APPROVED** [review]

**Pull request merged**